cmd/codeqlExecuteScan.go

```diff
@@ -185,13 +187,51 @@ func uploadResults(config *codeqlExecuteScanOptions, repoInfo RepoInfo, token st
 
 	//if no git pramas are passed(commitId, reference, serverUrl, repository), then codeql tries to auto populate it based on git information of the checkout repository.
 	//It also depends on the orchestrator. Some orchestrator keep git information and some not.
+
+	var buffer bytes.Buffer
+	utils.Stdout(&buffer)
 	err := execute(utils, cmd, GeneralConfig.Verbose)
 	if err != nil {
 		log.Entry().Error("failed to upload sarif results")
-		return err
+		return "", err
 	}
+	utils.Stdout(log.Writer())
+
+	url := buffer.String()
+	return strings.TrimSpace(url), nil
```

Comment:
@daskuznetsova try running upload with verbose and non-verbose mode. may be you might have to parse when verbose is enabled.

Reply:
in both modes output is the same - link to the sarif file